setup.py

```diff
@@ -17,7 +17,15 @@
     'Operating System :: MacOS :: MacOS X',
     'Operating System :: Microsoft :: Windows',
     'Operating System :: POSIX',
```

Comment:
```suggestion
```
This is probably not needed alongside `OS Independent`

Reply:
Indeed, but it is more verbose to leave them IMO.